### DISCUSSION TITLE
Order of elements in DisjointLoader

### DISCUSSION THREAD

**Discussion by jadlover** [danielegrattarola/spektral#406] [Q&A]
Hi, 

I want to get output of an intermediate layer for individual test examples to pass to UMAP. I have managed to get the output, but I am unsure of the index of the data point the output corresponds to. I am using DisjointLoader to load my test data (variable name data_test created using the Dataset function).
If using loader_test = DisjointLoader(data_test), will the nth output I record correspond to the nth entry in data_test, or is the data shuffled when passed to DisjointLoader? 

I sincerely appreciate any help!
Thanks!

**Comment by jadlover** [comment]
Oh! Never mind. I am stupid. Just change the shuffle variable. Yikes!

**Reply by danielegrattarola** [reply]
Sorry for not replying to this, Github messed up my notification settings. Anyway, glad you solved it!
